cms/static/js/views/settings/main.js

```diff
@@ -367,12 +368,18 @@ var DetailsView = ValidatingView.extend({
             message = this.post_enrollment_email_elem.val();
         }
 
-        $.post("/settings/send_test_enrollment_email",
+        validation = ViewUtils.keywordValidator.validate_string(message);
+        if (!validation.is_valid) {
+            message = gettext('There are invalid keywords in your email. Please check the following keywords and try again:\n');
```

Comment:
My bad... I know I asked you to pull the `\n` up here, but looking at it now, I realize we shouldn't be passing a newline (or any formatting markers) on to translation functions.

Can you revert this and add the `\n` in the line below? Sorry about that.